homeassistant/components/gree/switch.py

```diff
@@ -12,6 +16,29 @@
 from .const import COORDINATORS, DISPATCH_DEVICE_DISCOVERED, DISPATCHERS, DOMAIN
 from .entity import GreeEntity
 
+GREE_SWITCHES: tuple[SwitchEntityDescription, ...] = (
+    SwitchEntityDescription(
+        icon="mdi:lightbulb",
+        name="Panel light",
```

Comment:
The name is part of unique id. It cannot be changed.
```suggestion
        name="Panel Light",
```